pl_bolts/models/detection/__init__.py

```diff
@@ -1,8 +1,5 @@
-__all__ = []
-
 try:
-    from pl_bolts.models.detection.faster_rcnn import FasterRCNN  # noqa: F401
+    from pl_bolts.models.detection import components  # noqa
+    from pl_bolts.models.detection.faster_rcnn import FasterRCNN  # noqa
 except ModuleNotFoundError:  # pragma: no-cover
     pass  # pragma: no-cover
```

Comment:
I think this `try/except` clause seems unnecessary because
```python
    from pl_bolts.models.detection import components  # noqa
    from pl_bolts.models.detection.faster_rcnn import FasterRCNN  # noqa
```
won't raise `ModuleNotFoundError` even when any optional packages are installed.

Reply:
Also, let's replace `noqa` with `noqa: F401`.